manifests/0000_90_openshift-controller-manager-operator_01_prometheusrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  # TODO this should be a clusterrole
```

Comment:
@brancz is there a clusterrole we can bind in each namespace?

Reply:
I don’t believe we have that, but it would make sense to add, I agree.